.ci/setup_env_centos.sh

```diff
@@ -18,48 +18,42 @@ else
   centos_version=$(grep VERSION_ID /usr/lib/os-release | cut -d '"' -f2)
 fi
 
+[ "$centos_version" == 8 ] || die "This script is for CentOS 8 only"
+
 # Send error when a package is not available in the repositories
 echo "skip_missing_names_on_install=0" | sudo tee -a /etc/yum.conf
 
 # Check EPEL repository is enabled on CentOS
 if [ -z "$(yum repolist | grep 'Extra Packages')" ]; then
-	echo >&2 "ERROR: EPEL repository is not enabled on CentOS."
+	info "EPEL repository is not enabled on CentOS. Going to enable it."
 	# Enable EPEL repository on CentOS
-	sudo -E yum install -y wget rpm
-	wget https://dl.fedoraproject.org/pub/epel/epel-release-latest-${centos_version}.noarch.rpm
-	sudo -E rpm -ivh epel-release-latest-${centos_version}.noarch.rpm
+	sudo -E rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
 fi
 
 # Enable priority to CentOS Base repo in order to
 # avoid perl updating issues
-if [ "$centos_version" == "8" ]; then
-	repo_file=""
-	if [ -f /etc/yum.repos.d/CentOS-Base.repo ]; then
-		repo_file="/etc/yum.repos.d/CentOS-Base.repo"
-	elif [ -f /etc/yum.repos.d/CentOS-Linux-BaseOS.repo ]; then
-		repo_file="/etc/yum.repos.d/CentOS-Linux-BaseOS.repo"
-	else
-		die "Unable to find the CentOS base repository file"
-	fi
-	sudo echo "priority=1" | sudo tee -a "$repo_file"
-	sudo -E yum -y clean all
+repo_file=""
+if [ -f /etc/yum.repos.d/CentOS-Base.repo ]; then
+	repo_file="/etc/yum.repos.d/CentOS-Base.repo"
+elif [ -f /etc/yum.repos.d/CentOS-Linux-BaseOS.repo ]; then
+	repo_file="/etc/yum.repos.d/CentOS-Linux-BaseOS.repo"
+else
+	die "Unable to find the CentOS base repository file"
 fi
+sudo echo "priority=1" | sudo tee -a "$repo_file"
+sudo -E yum -y clean all
 
 echo "Update repositories"
 sudo -E yum -y update
 
-if [ "$centos_version" == "8" ]; then
-	echo "Enable PowerTools repository"
-	sudo -E yum install -y yum-utils
-	sudo yum-config-manager --enable powertools
-fi
+echo "Enable PowerTools repository"
+sudo -E yum install -y yum-utils
+sudo yum-config-manager --enable powertools
 
 echo "Install chronic"
 sudo -E yum -y install moreutils
 
-if [ "$centos_version" == "8" ]; then
-	chronic sudo -E yum install -y pkgconf-pkg-config
-fi 
+chronic sudo -E yum install -y pkgconf-pkg-config
```

Comment:
Can you also drop this chronic usage? :-)
It's not consistent across this file.

Reply:
@fidencio  good point.

I saw somewhere a discussion regarding whether to remove the chronic or not from kata scripts. So I think it's better to keep it until we don't have an agreement (or do we already have one?).